aodn_cloud_optimised/config/dataset/radar_velocity_hourly_averaged_delayed_qc_main.json

```diff
@@ -169,7 +168,25 @@
   "schema_transformation": {
     "global_attributes": {
       "set": {
-        "title": ""
+        "project": "Integrated Marine Observing System (IMOS)",
+        "Conventions": "CF-1.6,IMOS-1.4",
+        "abstract": "The IMOS Ocean Radar Facility is producing NetCDF files containing quality controlled vector current maps at 1 hour time interval. They are produced from radial currents, which represent the surface sea water current component along the radial direction from a receiver antenna. Radials are calculated from the shift of the Bragg peaks in a power spectrum. They are mapped onto specific angles through a Direction-Finding algorithm and are converted into current vector components using an unweighted least-squares fit. Radials and vector maps are computed using the standard software provided by the ocean radar manifacturer, CODAR Ocean Sensors (COS). Radials are extracted from the Doppler spectra using the calibrated antenna pattern. Thresholds for radial SNR and velocities are set to 6dB, and 150 cm s-1. Vectors are produced at each grid point using an unweighted least-squares fit. Radial velocities from the two sites falling within a search radius R=20 km around each grid point are used in the computation. At least two radials per site are required in computation of the velocity components. QC procedures apply first to intersection angle (GDOP>=30 & GDOP<=150); the U, V current components are then further inspected using 4 different statistical methods: absolute deviation from the median (MAD); statistics of the velocity distributions; statistics of the distributions of the 1st derivative; statistics of the distributions of the high-frequency components. Data are flagged based on the results of the statistical tests: 4, if three or more tests fail; 3, if two tests fail; 2, if one test fails; 1, no test fails. The final product is produced on a regular geographic grid. More information on the data processing is available through the document: Quality Control procedures for ACORN radars Manual Version 3.0 Integrated Marine Observing System. DOI: 10.26198/5c89b59a931cb (http://dx.doi.org/10.26198/5c89b59a931cb)",
+        "author": "Cosoli, Simone; Hetzel, Yasha",
+        "author_email": "simone.cosoli@uwa.edu.au; yasha.hetzel@uwa.edu.au",
+        "institution": "IMOS Ocean Radar Facility",
+        "institution_references": "https://imos.org.au/facility/ocean-radar",
+        "data_centre": "Australian Ocean Data Network (AODN)",
+        "data_centre_email": "info@aodn.org.au",
+        "instrument": "CODAR Ocean Sensors/SeaSonde",
+        "source": "Terrestrial HF radar",
+        "file_version": "Level 1 - Quality Controlled data",
+        "geospatial_lat_units": "degrees_north",
```

Comment:
we can keep but for future ref this is not important, as it was only used to exaplin the geospatial_lat_min ... variables

Reply:
updated in f0744c2